test/integration/targets/copy/tasks/main.yml

```diff
@@ -174,7 +174,7 @@
 
 - name: Copy with mode=preserve
   copy:
-    src: '{{ output_file }}'
+    src: 'foo.txt'
     dest: '{{ output_dir }}/copy-foo.txt'
```

Comment:
This is incorrect.  We have to transfer from ``` { output_file }}``` becaue we've set the mode of that file to 0547.